llm-tracing/requirements.txt

```
@@ -0,0 +1,16 @@
+fastapi==0.115.11
+fastapi[standard]
```

This does not seem correct.

```suggestion
fastapi[standard]==0.115.11
```